### PR TITLE
Integrate GPO information into the direct AD integration proc

### PR DESCRIPTION
**Pull request by asteflova**
The assembly on direct AD integration includes instructions on configuring GPOs in the form of an IMPORTANT admonition box. We can turn it into a regular procedure step instead and throw in a reference to sssd-ad(5) for good measure. Confirmed in https://github.com/theforeman/foreman-documentation/pull/3149#discussion_r1688116978.

* [x] I am okay with my commits getting squashed when you merge this PR.
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.11/Katello 4.13
* [x] Foreman 3.10/Katello 4.12
* [x] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8/6.9)
* [x] Foreman 3.8/Katello 4.10
* [x] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [x] Foreman 3.6/Katello 4.8
* [x] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* We do not accept PRs for Foreman older than 3.5.

### CODE REVIEW

**Review by Lennonka — APPROVED** [review]
LGTM

File: guides/common/modules/proc_configuring-direct-ad-integration-with-gss-proxy.adoc [hunk 1]
```diff
@@ -59,6 +59,26 @@ euid = __ID_of_Apache_User__
 # chown root.{apache-user} /etc/httpd/conf/http.keytab
```
Comment (apinnick) [inline guides/common/modules/proc_configuring-direct-ad-integration-with-gss-proxy.adoc:59]: FYI, since I realize this is outside the scope of your PR. [RHSSG](https://redhat-documentation.github.io/supplementary-style-guide/#commands-in-code-blocks) says to use only one command per code block.

File: guides/common/modules/proc_configuring-direct-ad-integration-with-gss-proxy.adoc [hunk 2]
```diff
@@ -59,6 +59,26 @@ euid = __ID_of_Apache_User__
...
 # chmod 640 /etc/httpd/conf/http.keytab
 ----
+. Configure SSSD to map Group Policy Objects (GPOs) from AD to the `foreman` PAM service:
```
[the remark this answers is not in the document]
Comment (asteflova) [reply guides/common/modules/proc_configuring-direct-ad-integration-with-gss-proxy.adoc]: Your feedback took me on a wonderful tour of three different SSSD man pages and I enjoyed every moment of it. I rephrased the step to make it more fitting while making sure acronyms are spelled out on first occurrence.

File: guides/common/modules/proc_configuring-direct-ad-integration-with-gss-proxy.adoc (hunk 1 above)
Comment (asteflova) [reply guides/common/modules/proc_configuring-direct-ad-integration-with-gss-proxy.adoc:59]: I actually didn't know that, thank you. But yes, let's not worry about it now, I'm pretty sure we'll need to revisit this procedure anyway and things might look quite different in the end.

**Comment by apinnick** [comment]
LGTM

**Comment by asteflova** [comment]
Merged to "master" and cherry-picked:

883fcc8cf7..a849a427f4  3.11 -> 3.11

Conflicts on branches 3.10 and below, I'm not cherry-picking there.
